### ISSUE TITLE
Add 2020 vs 2024 election voter turnout deployment

### ISSUE THREAD

**Issue by Smorgan05** [john-guerra/US_Elections_Results#6]
It would be really great to be able to compare overall voter turnout between the elections on the county level.

**Comment by john-guerra** [comment]
Yeah, I need to get the data first. If you find an official open data
source, please let me know

John Alexis Guerra Gómez
On my phone, brevity and all

**Comment by Smorgan05** [comment]
I can pull them from the state websites to start consolidating them.  Once the results are finalized then we can use that.

**Comment by Smorgan05** [comment]
[Turnout_2024G_v0.3.csv](https://github.com/user-attachments/files/18579699/Turnout_2024G_v0.3.csv)

University of Florida has put together a state by state voter turnout analysis with website links to gather the data at the county level. 

https://election.lab.ufl.edu/dataset/2024-general-election-turnout-rates-v0-3/

**Comment by john-guerra** [comment]
Thanks @Smorgan05 please see here for my visualization including 2024 election results https://github.com/john-guerra/USPresidentialElectionVisualization
